app/models/censor_rule.rb

```diff
@@ -44,6 +53,10 @@ class CensorRule < ApplicationRecord
           public_body_id: nil)
   }
 
+  cattr_accessor :canned_replacements,
+                 instance_writer: false,
+                 default: DEFAULT_CANNED_REPLACEMENTS.dup
```

Comment:
Could add a spec in `alavetelitheme` showing these are customisable

Reply:
Hmm yeah, not sure where the best place to document this is. Going to add [more like it](https://github.com/mysociety/alaveteli/issues/6699) so will consider in future when we have a bit more idea of where's appropriate.